src/components/Book.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import '../App.css';
 
 function Book(props) {
```

Comment:
### [Optional]
- It would be better if we use ES6 syntax instead of the function keyword. We are using `const` which is ES6 so it would be better if we follow with the consistency